### ISSUE TITLE
[Suggestion] Add virtual "aggregated" stations

### ISSUE THREAD

**Issue by ndrpnt** [juliuste/direkt.bahn.guru#13]
Hi, I'd love to display the direct connections from a city, as opposed to a single station. I.e, have a "Berlin (any station)" option in the search bar.

**Comment by juliuste** [comment]
I agree that this feature makes sense, however this is not really trivial to get working with the existing datasource(s), especially for the autocompletion. Will try to think about possible solutions over the next few days, feel free to ping me again if I forget 😄

**Comment by AghiadHaloul** [comment]
I was thinking about a similar suggestion as well :)

Maybe instead of having a city as a destination, the user could write type in two(+) stations, and then the results from both stations are merged and displayed (duplicates results could show the shorter time).

This is helpful for "twin-cities" situations likes Ulm(BaWü) and Neu-Ulm(Bayern) laying on opposite sides of the river, Mannheim/Ludwigshafen, as well as small towns have different closest-train-station-30-mins-away (Harz mountains).
